website/code_issues.qmd

```diff
@@ -319,3 +319,44 @@ Calls of `rm(list = ls())` to remove variables of the current environment, shoul
 
 :::
 
+
+
+
+---------------------------------------------
+
+
+# Calling `installed.packages()`
+
+## Problem
+
+You are calling `installed.packages()` to check if a package is installed.
+
+## Solution
+
+Instead of using `installed.packages()`, use `requireNamespace("pkg")` or `require("pkg")` to find out if packages are available.
+
+### Details
+
+::: {.callout-note title="CRAN Review Communication" appearance="simple" icon=false collapse=true}
+
+You are using `installed.packages()` in your code. As mentioned in the notes of `installed.packages()` help page, this can be very slow. Therefore do not use `installed.packages()`.
```

Comment:
Would it be helpful to link to this help page? https://stat.ethz.ch/R-manual/R-devel/library/utils/html/installed.packages.html

Reply:
I added it in the first paragraph of the "Details"